src/main/java/org/junit/runners/model/InvalidTestClassError.java

```diff
@@ -0,0 +1,41 @@
+package org.junit.runners.model;
+
+import java.util.List;
+
+/**
+ * Thrown by {@link org.junit.runner.Runner}s in case the class under test is not valid.
+ * <p>
+ * Its message conveniently lists all of the validation errors.
+ *
+ * @since 4.13
+ */
+public class InvalidTestClassError extends InitializationError {
+    private static final long serialVersionUID = 1L;
+
+    private final Class<?> testClass;
```

Comment:
I don't think we have to store `testClass` since it's only used in the constructor.